app/assets/javascripts/dashboard/protocols.js.coffee

```diff
@@ -167,6 +167,13 @@ $(document).ready ->
             $('.service-requests-table').on 'all.bs.table', ->
               $(this).find('.selectpicker').selectpicker()
 
+      $(document).on 'click', '.dropdown-menu li', ->
```

Comment:
Is it necessary that the adjacent text field is disabled when you don't select an option from the select?